faunadb-scala/src/main/scala/faunadb/query/package.scala

```diff
@@ -691,11 +691,19 @@ package object query {
   /**
     * An HasIdentity expression.
     *
-    * '''Reference''': [[https://app.fauna.com/documentation/reference/queryapi#authentication]]
+    * '''Reference''': [[https://docs.fauna.com/fauna/current/api/fql/functions/hasidentity]]
     */
+  @deprecated("use HasCurrentIdentity instead", "4.0.0")
   def HasIdentity(): Expr =
     Expr(ObjectV("has_identity" -> NullV))
 
+  /**
+    * An HasCurrentIdentity expression.
+    *
+    * '''Reference''': [[https://docs.fauna.com/fauna/current/api/fql/functions/hascurrentidentity]]
+    */
+  def HasCurrentIdentity(): Expr =
+    Expr(ObjectV("has_current_identity" -> NullV))
   // String Functions
```

Comment:
An empty space would be nice here :)

Reply:
https://github.com/fauna/faunadb-jvm/pull/255/commits/2cd345fb8ca2e1aadf905f7da2494c544f12e86e

Reply:
that commit dismissed the reviews :sob: